XPlat/TensorFlow.Lite/source/Xamarin.TensorFlow.Lite.Bindings.XamarinAndroid/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="NuGet.Build.Packaging" version="0.2.0" targetFramework="monoandroid81" developmentDependency="true" />
```

Comment:
What's this one used for?  Is it so you can `/t:Pack` ? Please switch this to using `PackageReference` and we should even consider using the new SDK style project with the MSBuild Extensions.  Check out the Android Things projects if you want to see an example of how to do this with android binding projects: 
https://github.com/xamarin/XamarinComponents/blob/master/Android/AndroidThings/source/driver-apa102/driver-apa102.csproj